src/components/DatePicker/DatePicker.st.css

```diff
@@ -0,0 +1,581 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback, font, zeroAsTrue;
+}
+
+:import {
+    -st-from: "wix-style-react/dist/src/Calendar/Calendar.st.css";
+    -st-default: WSRCalendar;
+    -st-named: dayCircle; /* currently due to a stylable issue this is the best way to override styling of nested classes */
+}
+
+:import {
+    -st-from: "wix-ui-core/dist/src/components/popover/Popover.st.css";
+    -st-named: popoverContent;
+}
+
+/* Defaults - default params */
+:vars {
+    calendarWidth: 280px;
+    calendarHeight: 332px;
+    sidePadding: 20px;
+
+    DefaultBackgroundColor: color-1;
+
+    DefaultNavigationArrowsColor: color-5;
+    DefaultMonthAndYearCaptionFont: "{theme: 'Body-M', lineHeight: '24px'}";
+    DefaultMonthAndYearCaptionColor: color-5;
+
+    DefaultWeekdayFont: "{theme: 'Body-M', lineHeight: '20px'}";
+    DefaultWeekdayTextColor: color-8;
+
+    DefaultDayFont: "{theme: 'Body-M', lineHeight: '20px'}";
+    DeafultDayColor: color-5;
+    DefaultSelectedDayColor: color-1;
+    DefaultSelectedDayBGColor: color-8;
+    DefaultSelectedDayBorderRadius: 50%;
+    DefaultDisabledDayColor: color-3;
+}
+
+/*Overrides*/
+:vars {
+    /*
+        The calendar background color
+        @default color-1
+    */
+    BackgroundColor: --overridable;
+    /*
+        The color of the calendar navigation arrows
+        @default color-5
+    */
+    NavigationArrowsColor: --overridable;
+    /*
+        The font of the calendar month snd year caption. The font size is 16px and could not be overridden.
+        @default {theme: 'Body-M', lineHeight: '24px'}
+    */
+    MonthAndYearCaptionFont: --overridable;
+    /*
+        The color of the calendar month and year caption
+        @default color-5
+    */
+    MonthAndYearCaptionColor: --overridable;
+    /*
+        The font of the calendar's weekdays section. The font size is 14px and could not be overridden.
+        @default {theme: 'Body-M', lineHeight: '20px'}
+    */
+    WeekdayFont:  --overridable;
+    /*
+        The font color of the calendar's weekdays.
+        @default color-8
+    */
+    WeekdayTextColor: --overridable;
+    /*
+        The font of a specific date. The font size is 14px and could not be overridden.
+        @default {theme: 'Body-M', lineHeight: '20px'}
+    */
+    DayFont: --overridable;
```

Comment:
From what I understood from Leigh, this should be overridable, no?

Reply:
yes, so we have just added a task that includes it:)